types/office-js-preview/index.d.ts

```diff
@@ -10938,53 +10938,65 @@ declare namespace Office {
          */
         getInitializationContextAsync(callback: (asyncResult: Office.AsyncResult<string>) => void): void;
         /**
-         * Asynchronously gets the ID of a saved item.
+         * Asynchronously gets the {@link https://learn.microsoft.com/exchange/client-developer/exchange-web-services/ews-identifiers-in-exchange | Exchange Web Services (EWS) item identifier}
+         * of a saved item.
          *
          * When invoked, this method returns the item ID via the callback function.
          *
-         * **Note**: If your add-in calls `getItemIdAsync` on an item in compose mode (e.g., to get an `itemId` to use with EWS or the REST API),
-         * be aware that when Outlook is in cached mode, it may take some time before the item is synced to the server.
-         * Until the item is synced, the `itemId` is not recognized and using it returns an error.
-         *
          * @remarks
          * [Api set: Mailbox 1.8]
          *
          * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read item**
          *
          * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Appointment Organizer
          *
+         * **Important**:
+         *
+         * - The item ID returned isn't identical to the Outlook Entry ID or the ID used by the Outlook REST API.
```

Comment:
Is it "Entry" or "Entra"?

Reply:
Entry ID. See https://learn.microsoft.com/office/vba/api/outlook.mailitem.entryid.